CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 cmake_minimum_required(VERSION 3.16.3)
 
 if(NOT CMAKE_CXX_STANDARD)
-  set(CMAKE_CXX_STANDARD 14) # Supported values are ``14``, ``17`` and ``20``.
+  set(CMAKE_CXX_STANDARD 17) # Supported values are ``14``, ``17`` and ``20``.
```

Comment:
Best would be to remove `14` from the comment as well.

Reply:
Good catch!